components/dashboard/src/admin/TeamDetail.tsx

```diff
@@ -64,8 +103,43 @@ export default function TeamDetail(props: { team: Team }) {
                 </div>
             </div>
             <div className="flex mt-6">
-                {!team.markedDeleted && teamMembers && <Property name="Members">{teamMembers.length}</Property>}
-                {!team.markedDeleted && <Property name="BillingMode">{billingMode?.mode || "---"}</Property>}
+                {!team.markedDeleted && <Property name="Members">{teamMembers?.length || "?"}</Property>}
+                {!team.markedDeleted && <Property name="Billing Mode">{billingMode?.mode || "---"}</Property>}
+                <Property name="Billing Strategy" actions={[]}>
```

Comment:
no, in UBP we have different ways (strategies) of renewing the billing cycle. currently stripe and other (free)

Reply:
Same as below in https://github.com/gitpod-io/gitpod/pull/14594#discussion_r1028109269:

> It was also more confusing because I thought that a stripe user ID would be created always when a user was using UBP, regardless if they are using the Free plan or upgraded their plan.

Thanks for clarifying! 🥯